fact/src/event/mod.rs

```diff
@@ -89,6 +89,7 @@ impl From<Event> for fact_api::FileActivity {
             file: Some(file),
             timestamp: Some(timestamp),
             process: Some(process),
+            node: value.hostname.to_string(),
```

Comment:
Actually, why is this not `node: Some(value.hostname.to_string())`? All fields in protobuf definitions are nullable, is it because the field is a straight string and protobuf uses empty for its default instead?